### PR TITLE
make technologies and endpoints collapsable

### PR DESCRIPTION
**Pull request by Briahana-Mauge**
## Description

Updates the readme to include collapsible technology and endpoint sections. Alejo suggested to include these to shorten the ReadMe.

### CODE REVIEW

**Comment by joseph-p-pasaoa** [comment]
Omg Github can collapse and expand markdown... let me pick up my brain off the floor. This is great!

File: README.md [hunk 1]
```diff
@@ -30,13 +30,16 @@
 
 
 ## I. Technologies Implemented
-- SendGrid API v3
-- Google Calendar
-
++ SendGrid API v3
```
Comment (joseph-p-pasaoa) [inline README.md:33]: @Briahana-Mauge I had these two separated because they aren't built into our app aka external techs. But the change is fine with me.

Comment (Briahana-Mauge) [reply README.md]: I put the space back between the two!  Should I add Chartjs to that group as well?

Comment (joseph-p-pasaoa) [reply README.md]: Chartjs was imported in so no I don't think so.

**Pull request merged**